backend/open_webui/routers/supershot.py

```
@@ -0,0 +1,111 @@
+                response.content,
+                status_code=response.status,
+                headers=dict(response.headers),
+                background=BackgroundTask(
+                    cleanup_response, response=response, session=session
+                ),
+            )
+        else:
+            # For non-streaming responses, get JSON and return it
+            try:
+                response_data = await response.json()
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)